.gitignore

```diff
@@ -1,9 +1,58 @@
-.DS_Store
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+
+# exposed public scripts except sw.js
+/public/scripts/*
+!/public/scripts/sw.js
+
+# env
+.env/.env.dev.local
+
+# testing
+/coverage
 
-node_modules
+# build
+/build
+/build-development
+/build-demo
+/build-staging
+/build-mobile-staging
+/build-ci
+/build-production
```

Comment:
so for this .gitignore, i thinkk most of this stuff got moved to the packages/web and packages/mobile sections. mind double checking on that?

Reply:
Yea for sure, I had things like the builds and the public scripts top level showing up in my git diff so I just copied over another gitignore that we use. I can give it a once over and filter it down 👍

Reply:
I think this happens when switching from `master` to `main`. Git doesn't delete those files for some reason even though they don't exist on `main`. The better solution is to manually delete those files (since we won't be using `master` anymore). That way we don't need to change the gitignore

Reply:
good point, I'll just remove and revert the gitignore 👍